### PR TITLE
[SMALLFIX] Revert "delete two site properties""

### PR DESCRIPTION
**Pull request by peisun1115**
This reverts commit 2dd07bc4d9c8c589c1a437aff3179c172d6e3fa1.

### CODE REVIEW

**Comment by peisun1115** [comment]
@apc999

File: tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ public RemoteBlockInStreamIntegrationTest(String dataServer, String transferType
     mLocalAlluxioClusterResource = new LocalAlluxioClusterResource.Builder()
         .setProperty(PropertyKey.WORKER_DATA_SERVER_CLASS, dataServer)
         .setProperty(PropertyKey.WORKER_NETWORK_NETTY_FILE_TRANSFER_TYPE, transferType)
+        .setProperty(PropertyKey.USER_BLOCK_REMOTE_READER_CLASS, reader)
```
Comment (apc999) [inline tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java]: I think we can remove this change and the next change , right?
just to keep them in doc and in supported keys. not necessary to reference them

**Comment by apc999** [comment]
one comment left. otherwise LGTM

File: tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java (hunk 1 above)
Comment (peisun1115) [inline tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java]: good point. done.

**Comment by apc999** [comment]
LGTM

**Pull request merged**